.github/workflows/js-linting.yml

```diff
@@ -0,0 +1,43 @@
+name: JavaScript Linting
+
+on:
+  push:
+    branches:
+      - trunk
+    paths:
+      - "**.js"
+      - "**.mjs"
+      - .github/workflows/js-linting.yml
+  pull_request:
+    paths:
+      - "**.js"
+      - "**.mjs"
+      - .github/workflows/js-linting.yml
```

Comment:
💅  Maybe running this WorkFlow on PR against trunk would be enough?

Reply:
If the linting check only runs on PR against trunk, PR to PR, like this current one (#34), wouldn't be run. I think finding out linting errors earlier should be ideal.

Reply:
What I meant is to remove the `push` directive.

Reply:
The temporarily merged code of a PR to be run checks is not always the same as the merged one after a PR actually gets merged.

For example, if there are other commits are pushed to trunk after this PR was opened, this PR won't be triggered and run the check again if there are no conflicts. When two PR change the same file like this:

```diff
const config = {
+	hello: '123', // Added by PR A
	howdy: '000',
+	hello: '456', // Added by PR B
 };
};
```

Both PR could pass the check, but it would be a problem after merging. When this happens, the check run on push events of trunk might find out this problem.

Reply:
Right! Thanks for the explanation 👌